benchflow-dsl/src/main/scala/cloud/benchflow/dsl/explorationspace/ExplorationSpaceGenerator.scala

```diff
@@ -11,30 +11,41 @@ import scala.collection.mutable
  */
 object ExplorationSpaceGenerator {
 
+  // types for easier readability
   type ServiceName = String
   type VariableName = String
   type VariableValue = String
   type NumUsers = Int
   type DimensionLength = Int
   type Index = Int
 
-  case class ExplorationSpace(
+  case class ExplorationSpaceDimensions(
     users: Option[List[NumUsers]],
     memory: Option[Map[ServiceName, List[Bytes]]],
```

Comment:
I wanted remove the nesting so it is more clear and to avoid having to access another layer